portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,282 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
-}
+    margin-left: auto;
+    margin-right: auto;
+    width: 1000px;
+  }
+  #whiteCenter {
```

[4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation): Separate rules by new lines.

```suggestion
}

#white-center {
```

Here and elsewhere.